src/ore/src/metrics/third_party_metric.rs

```diff
@@ -82,23 +72,3 @@ impl<T: MetricVecBuilder> ThirdPartyMetric<MetricVec<T>> {
             .expect("creating a metric from values")
     }
 }
-
-impl<P: Atomic> ThirdPartyMetric<GenericCounterVec<P>> {
-    /// Creates a delete-on-drop counter that can be scraped by third parties.
-    pub fn get_third_party_delete_on_drop_counter<'a, L: PromLabelsExt<'a>>(
-        &self,
-        labels: L,
-    ) -> DeleteOnDropCounter<'a, P, L> {
-        self.inner.get_delete_on_drop_counter(labels)
-    }
-}
-
-impl<P: Atomic> ThirdPartyMetric<GenericGaugeVec<P>> {
-    /// Creates a delete-on-drop gauge that can be scraped by third parties.
-    pub fn get_third_party_delete_on_drop_gauge<'a, L: PromLabelsExt<'a>>(
-        &self,
-        labels: L,
-    ) -> DeleteOnDropGauge<'a, P, L> {
-        self.inner.get_delete_on_drop_gauge(labels)
-    }
-}
```

Comment:
The two impls should be used at some point, better to keep I think.

Reply:
thanks for the review! re-added